docs/swiper.md

```diff
@@ -0,0 +1,500 @@
+---
+id: swiper
+title: Swiper
+---
+
+### Usage
+
+```js
+import { Swiper } from 'react-native-elements'
+
+<Swiper>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  {/* ... */}
+</Swiper>
+
+<Swiper
+  vertical {/* slide up / down instead left / right */}
```

Comment:
I would change this to `direction`, enum being either `horizontal` | `vertical`

Reply:
Ohhh... [It was so](https://github.com/oxyii/react-native-web-swiper/tree/3517a7f69be3bce835e75876004b520efbbefb77#properties) in the initial version. Ok, will change back